index.html

```diff
@@ -1,67 +1,218 @@
+                        <div class="paragraphworks">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</div>
+                        <div>
+                            <ul class="boxes2">
+                                <li class="waks1"><a href="#"></a><img src="image/html2.png" alt="HTML"></li>
+                                <li class="waks2"><a href="#"></a><img src="image/bootstrap2.png" alt="Bootstrap"></li>
+                                <li class="waks3"><a href="#"></a><img src="image/rubby2.png" alt="Ruby"></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <button class="button2">See Project</button>
+                </div>
+            </div>
         </div>
 
-        <div class="group">
-            <img src="image/Group.png" alt="Group">
+        <div class="afterworks">
+            <div class="immediate-about">
+                <div class="aboutme">About Me</div>
+                <div class="aboutpara">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</div>
+                <button class="aboutbutton"></button>
+            </div>
+            <div class="group2"></div>
+            <div class="central-line"><img src="image/centeralignment.png" alt="Central Line"></div>
+            <div>
+                <div class="language-container">
+                    <img src="image/diamond.png" alt="Diamond">
+                    Language
+                </div>
+                <ul class="language-list">
+                    <li class="javascript">JavaScript</li>
+                    <li class="ruby">Ruby</li>
+                    <li class="html">HTML</li>
+                    <li class="css">CSS</li>
+                </ul>
+                <div class="frames-containerwork">
+                    <img src="image/square.png" alt="Square">
+                    Frameworks
+                </div>
+                <ul class="frameworks-list">
+                    <li class="react">React.js</li>
+                    <li class="ror">Ruby On Rails</li>
+                    <li class="rspec">RSpec</li>
+                    <li class="capybara">Capybara</li>
+                </ul>
+                <div class="skills-container">
+                    <img src="image/circle.png" alt="Circle">
+                    Skills
+                </div>
+                <ul class="skills-list">
+                    <li class="database">Database Management</li>
+                    <li class="control">Version Control</li>
+                    <li class="cli">CLI</li>
+                    <li class="design">API Design</li>
+                    <li class="webdev">Web Development</li>
+                </ul>
+            </div>
+        </div>
```

Comment:
Great job on your section, I have a couple of suggestions to make it look even better and closer to the provided [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-27&t=ZJcVC8XUqegdTgGs-0):

- [ ] Instead of using a div with divs inside, try to create an `ul` with an `li` tag for each of your cards in the **Languages, Frameworks and Skills sections.** This will help make your code more semantic and easier to read.
```
Example (Just to give you an idea):
<ul>
  <li>
    <div>
       {img, title}
       {ul list}
    </div>
  </li>
  ...
</ul>
```
- [ ] Check the size of each image/icon for the list, as it doesn't seem to be aligned with the template. This will ensure that your design looks consistent with the template.
Keep up the good work! 🙌